ansible/module_utils/port_utils.py

```diff
@@ -301,3 +306,8 @@ def get_port_indices_for_asic(asic_id, port_name_list_sorted):
         port_index_map[val] = idx
 
     return port_index_map
+
```

Comment:
@lizhijianrd this main function is a little bit confusion, why we need a main function here?

Reply:
Good catch! I used it for test and forgot to remove it before commit. Deleted it in my latest commit.